### ISSUE TITLE
Support React 18

### ISSUE THREAD

**Issue by eggplants** [storybook-community/addon-queryparams#4]
`peerDependencyError` has happened when I try to install this package in my project using React 18. Need to update package dependencies?

**Comment by soobing** [comment]
You can set query params using JavaScript as follows:
```
const setQueryParams = (params) => {
  const searchParams = new URLSearchParams(params);
  window.history.replaceState({}, '', `${window.location.pathname}?${searchParams}`);
};
setQueryParams({ version: '2.0' });
```

In my case I added this code in decorator.
```
<Meta title="Select"
  component={Select}
  decorators={[
    Story => {
      const setQueryParams = (params) => {
        const searchParams = new URLSearchParams(params);
        window.history.replaceState({}, '', `${window.location.pathname}?${searchParams}`);
      };
      setQueryParams({ version: '2.0' });
      return <Story />
    },
  ]}/>
```

**Comment by eggplants** [comment]
This issue will be closed by #6
